setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
```

Comment:
`import pages` from `setup.py` won't work because you're trying to install `pages`. It's a 🐔 and 🍳 issue.